.eslintrc.js

```diff
@@ -12,6 +12,26 @@ module.exports = {
     ecmaVersion: 'latest',
     sourceType: 'module',
   },
-  plugins: ['react', '@typescript-eslint'],
-  rules: {},
+  plugins: ['react', 'react-native', '@typescript-eslint'],
+  rules: {
```

Comment:
These rules were determined because eslint was complaining about the way the ts starter code was set up